gamelogic.js

```diff
@@ -1,13 +1,67 @@
 function displayWordSoFar(word, guesses) {
-  // WRITE ME
+  //code to display "_ _ _ _" in accordance to the length of the word
+  // console.log(word);
+  // console.log(guesses);
+
+  // let array = [];
+  //
+  // for (let i = 0; i < word.length; i++) {
+  //   array.push('_ ');
+  //
+  // }
+  //
+  // let text = array.join('').toString();
+  // return text;
+
+  //code to display where the letters that are guessed so far appear on it
+  let output = "";
+  for( let i = 0; i < word.length; i++) {
+    if (guesses.includes(word[i])) {
+      output = output + word[i] + " ";
+    } else {
+      output = output + "_ ";
+    }
+  }
+  return output;
 }
 
 function isGameWon(word, guesses) {
-  // WRITE ME
+  let output = displayWordSoFar(word, guesses);
+  output = output.split(" ").join("");
+  console.log(output);
+
```

Comment:
Hmmm. Dit zorgt er wel voor dat de regel met geraden letters en underscores 2 keer wordt getoond in index.js. Is bovendien niet de allerhandigste oplossing, krijgt het woord al binnen
Je zou het zo kunnen doen:
   for (let i = 0; i < word.length; i++) {
        // if a character in word is not in guesses,
        // then return false
        const letter = word[i];

        if ( !guesses.includes(letter) ) {
            return false;
        }

    }
    // if we get here all characters in word have been guessed
    return true;